### DISCUSSION TITLE
Hoarder (read-it-later alternative now that Omnivore shut down)

### DISCUSSION THREAD

**Discussion by RayBB** [coollabsio/coolify#4078] [Service Template Requests]
https://docs.hoarder.app/Installation/docker

**Comment by ptrsmk** [comment]
This one seems pretty straight-forward.

**Comment by RayBB** [comment]
PR'd here https://github.com/coollabsio/coolify/pull/4120

**Comment by peaklabs-dev** [comment]
This will be available later today.
